src/vsr/replica.zig

```diff
@@ -4589,7 +4589,7 @@ pub fn ReplicaType(
 
                 const register_request = std.mem.bytesAsValue(
                     vsr.RegisterRequest,
-                    prepare.body()[0..@sizeOf(vsr.RegisterRequest)],
+                    prepare.body_used()[0..@sizeOf(vsr.RegisterRequest)],
```

Comment:
I didn't look at these; is there a reason we further slice by `sizeOf`? (on all the below too)

Reply:
No, probably just by habit.